packages/console/src/hooks/use-api.ts

```diff
@@ -37,15 +39,17 @@ const useApi = ({ hideErrorToast }: Props = {}) => {
       ky.create({
         hooks: {
           beforeError: conditional(
-            !hideErrorToast && [
-              (error) => {
-                const { response } = error;
+            hideErrorToast
```

Comment:
if conditional works as expected I would prefer conditional. Or you can remove conditional if ternary is used.

Reply:
Oh, I didn't realise there was a conditional... lol, my bad
I will remove the outer conditional, as it won't work in this case (had a discussion with simeng and sijie earlier on this).